pkg/detectors/readme/readme.go

```diff
@@ -20,7 +20,7 @@ var (
 	client = common.SaneHttpClient()
 
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
-	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"readme"}) + `\b([a-zA-Z0-9_]{32})\b`)
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"rdme_"}) + `\b([a-z0-9]{70})\b`)
```

Comment:
question: Do you know if they have stopped providing support for the older format? If so I think this is okay, but if there is still support (albeit deprecated) uses of the older key we would still want to make sure we detect those. So it might make sense to leave the older one and add a new regex for the new format.
Thoughts?

Reply:
A good way to do this is to add an OR in the regex

Reply:
Readme completely invalidated all previous keys and issued a new version of readme keys where the key looks like this -  "rdme_xn8s9hffe79257cd578c938791cffd1f429536d562a9e2927c30080bc87d45dbdc4c02". Notice the prefix  but the regex i suggested is not working on my machine. Sorry, not so well versed in Golang

here is the regex from gitleaks - https://github.com/zricethezav/gitleaks/blob/e35cb6707a97beb6d9495bceffec351d182c3c88/cmd/generate/config/rules/readme.go

Reply:
They had a security incident so they rotated all their keys and introduced new version of the keys

https://docs.readme.com/changelog/api-key-incident-may-2022

Reply:
was able to fix the regex but the verifyer is still not verifying the result.